src/components/app.jsx

```diff
@@ -129,64 +136,91 @@ const App = () => {
         }
     }, [user]);
 
+    useNotificationsListener(NotificationUrlKeys.GLOBAL_CONFIG, {
+        listenerCallbackMessage: (event) => {
+            const eventData = JSON.parse(event.data);
+            if (eventData.headers.messageType === 'announcement') {
+                if (
+                    announcementInfos != null &&
+                    announcementInfos.announcementId === eventData.headers.announcementId
+                ) {
+                    // If we receive a notification for an announcement that we already received we ignore it
+                    return;
+                }
+                const announcement = {
+                    announcementId: eventData.headers.announcementId,
+                    message: eventData.payload,
+                    severity: eventData.headers.severity,
+                    duration: eventData.headers.duration,
+                };
+                setAnnouncementInfos(announcement);
+            } else if (eventData.headers.messageType === 'cancelAnnouncement') {
+                setAnnouncementInfos(null);
+            }
+        },
+    });
```

Comment:
Please create memoized function for this to avoid remove/add listener each time App is rendered